### DISCUSSION TITLE
Publishing on the Snap store.

### DISCUSSION THREAD

**Discussion by aromatic-dev** [leozide/leocad#662] [Ideas]
@leozide would you be interested in publishing this on https://snapcraft.io?

**Comment by leozide** [comment]
What's the advantage of the snap store? We already have AppImage, FlatPak and .deb for distribution. I don't mind adding it if it's automatic and low maintenance.

**Reply by nathaneltitane** [reply]
> What's the advantage of the snap store? We already have AppImage, FlatPak and .deb for distribution. I don't mind adding it if it's automatic and low maintenance.

better visibility?

**Comment by aromatic-dev** [comment]
Indeed, as @nathaneltitane mentioned, for better visibility but also for ease of installation -- with linux distributions that come pre-configured with the snap store, it will be just a matter of clicking "install", or typing `sudo snap install leocad`.

**Comment by leozide** [comment]
It's here: https://snapcraft.io/leocad

**Reply by aromatic-dev** [reply]
Excellent work, thank you.

**Comment by aromatic-dev** [comment]
It's interesting see more and more distributions popping up here https://snapcraft.io/leocad under `
Users by distribution (log)` -- what do the numbers looking like?

**Reply by leozide** [reply]
A little over 1100 installs so far

**Reply by aromatic-dev** [reply]
Great, this is what I suspected it would be by now 😁
